src/app/legacy/providers/redeem-provider.tsx

```diff
@@ -1,8 +1,8 @@
+import { useQuery, useQueryClient } from '@tanstack/react-query'
```

Comment:
useQueryClient is probably unnecessary here, since <Providers> wraps this page into a <QueryClientProvider>